### PR TITLE
[MM-50349] Add Feature Flag to Disable Tour Tips

### PR DESCRIPTION
**Pull request by mylonsuren**
#### Summary

Adds a feature flag to disable tour tips for Rainforest testing.

- **Drafts Tour Tip**
image

- **RHS Viewing a Thread Tour Tip**
image

- **Post Actions Tour Tip**
image

*See https://community.mattermost.com/core/pl/b4pwu6jeb3n7pkejqcci5uy7fh for further context.*

#### Ticket Link

https://mattermost.atlassian.net/browse/MM-50349

#### Related PRs

- Has server changes https://github.com/mattermost/mattermost-server/pull/22263

#### Release Note

```release-note
NONE
```

### CODE REVIEW

**Review by AshishDhama — APPROVED** [review]

File: components/post/post_component.tsx [hunk 1]
```diff
@@ -116,6 +116,7 @@ export type Props = {
     isPostPriorityEnabled: boolean;
     isCardOpen?: boolean;
     shouldShowDotMenu: boolean;
+    tourTipsEnabled: boolean;
```
Comment (M-ZubairAhmed) [inline components/post/post_component.tsx:119]: where is this used here?

**Review by M-ZubairAhmed — APPROVED** [review]

**Review by lindy65 — APPROVED** [review]
Looks good to me 👍

**Comment by amyblais** [comment]
@AshishDhama @M-ZubairAhmed Let me know if you're able to help open a manual cherry-pick PR for ``release-7.8`` branch.

**Pull request merged**